docs/config.md

```diff
@@ -200,6 +200,27 @@ can override the default style with the following keys:
 ui.diff.format = "git"
 ```
 
+#### Color-words diff options
+
+In color-words diffs, changed words are displayed inline by default. Because
+it's difficult to read a diff line with many removed/added words, there's a
+threshold to switch to traditional separate-line format.
+
+* `max-inline-alternation`: Maximum number of removed/added word alternation to
+  inline. For example, `<added> <added>` sequence has 1 alternation, so the line
```

Comment:
Does `<added> <added>` make sense? Did you mean `<added> unchanged <added>`?

Reply:
Inserted ellipses, which I think can be read as repetition (plus unchanged if the user is aware of that.)